### ISSUE TITLE
[Proposal] Update dompdf/dompdf to 0.8.0 to fix PDF Invoice ErrorException using PHP 7.1

### ISSUE THREAD

**Issue by waitem** [laravel/cashier#386]
Laravel Spark v4.0.5 (using Laravel/Cashier 7.0.6) produces an error exception when trying to display an team invoice:

```
ErrorException in Table.php line 577:
A non-numeric value encountered
in Table.php line 577
at HandleExceptions->handleError(2, 'A non-numeric value encountered', '/home/vagrant/crimcram/vendor/dompdf/dompdf/src/FrameReflower/Table.php', 577, array('style' => object(Style), 'columns' => array(array('x' => 0, 'min-width' => 163.0, 'max-width' => 163.0, 'used-width' => null, 'absolute' => 163.0, 'percent' => 0, 'auto' => false), array('x' => 0, 'min-width' => 154.494, 'max-width' => 271.719, 'used-width' => null, 'absolute' => 0, 'percent' => 0, 'auto' => true)), 'i' => 1, 'dims' => array(0, 0, '0', '0', 'auto', 'auto', '2px'), 'delta' => 'auto')) in Table.php line 577
at Table->get_min_max_width() in Table.php line 407
at Table->reflow(object(Block)) in AbstractFrameDecorator.php line 801
at AbstractFrameDecorator->reflow(object(Block)) in Block.php line 738
at Block->reflow(object(Block)) in AbstractFrameDecorator.php line 801
at AbstractFrameDecorator->reflow(object(Block)) in Block.php line 738
at Block->reflow(null) in AbstractFrameDecorator.php line 801
at AbstractFrameDecorator->reflow() in Page.php line 133
at Page->reflow(null) in AbstractFrameDecorator.php line 801
at AbstractFrameDecorator->reflow() in Dompdf.php line 834
at Dompdf->render() in Invoice.php line 263
at Invoice->pdf
```

Cashier currently requires 0.7.x of dompdf/dompdf.

The problem appears to be resolved with v0.8.0 of dompdf/dompdf. See: [github issue](https://github.com/dompdf/dompdf/issues/1272)

This proposal is to update the version number of dompdf/dompdf in cashier's composer.json to ^0.8.0

**Comment by faustbrian** [comment]
@waitem Did you find a way to force install dompdf 0.8? Running into the same issue at the moment.

**Comment by waitem** [comment]
@faustbrian Sorry, no. Beyond my skill level I'm afraid! Hoping that either a) we get a new version of laravel/cashier, or b) someone else can show how to temporarily override the dompdf/dompdf version in laravel/cashier's composer.json until the new version comes out.

**Comment by faustbrian** [comment]
Just ended up cloning cashier locally and loading it that way instead of from packagist

**Comment by waitem** [comment]
@faustbrian Would you mind sharing how you configured your project to load the local cloned version, rather than the packagist one. Many thanks!

**Comment by waitem** [comment]
OK, I have seen that a fix has been made to the 7.0 branch a few days ago: [commit](https://github.com/laravel/cashier/commit/c595d24a1cba397f4dacdd6df6af874ac5c351c2)

Until a new 7.0.x release is made, I am able to resolve this issue by making the following to changes to composer.json:

```
    "require": {
         ...
          "laravel/cashier": "7.0.x@dev",
          ...
    }
   ...
    "repositories": [
        {
            "type": "git",
            "url": "https://github.com/laravel/cashier.git"
        }
    ]
```
